src/njoy21/input/COVR/Card3b/Hlibid/test/Hlibid.test.cpp

```diff
@@ -0,0 +1,29 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "COVR hlibid",
+  "[COVR], [Card3b], [Hlibid]"){
+
+  GIVEN( "valid inputs" ){
+    THEN( "the returned class has the correct value" ){
+      iRecordStream<char> iss( std::istringstream( "h" ) );
+      REQUIRE( "h" == argument::extract< COVR::Card3b::Hlibid >( iss ).value);
```

Comment:
Need to include a test where there are exactly 6 characters.

Reply:
The second test does just that.